pkg/web3/alchemy.go

```diff
@@ -108,3 +108,38 @@ func (a *AlchemyAPI) GetContractMetadata(contractID authgearweb3.ContractID) (*a
 
 	return &response, nil
 }
+
+func (a *AlchemyAPI) GetOwnersForCollection(contractID authgearweb3.ContractID) (*apimodel.GetOwnersForCollectionResponse, error) {
+	alchemyEndpoints, err := GetRequestEndpoints(a.Config.Alchemy, contractID.Blockchain, contractID.Network)
+	if err != nil {
+		return nil, err
+	}
+
+	if contractID.ContractAddress == "" {
+		return nil, fmt.Errorf("contractAddress is empty")
+	}
+
+	requestURL := alchemyEndpoints.NFTEndpoint
+	requestURL.Path = path.Join(requestURL.Path, "getOwnersForCollection")
+
+	requestQuery := requestURL.Query()
+	requestQuery.Set("contractAddress", contractID.ContractAddress)
+
+	requestURL.RawQuery = requestQuery.Encode()
+
+	log.Printf("Requesting owners for contractAddress: %s from network %s %s", contractID.ContractAddress, contractID.Blockchain, contractID.Network)
```

Comment:
This log is going to be noisy. Remove it, or use log factory which allows us to assign the log a INFO level.